build_projects/update-dependencies/project.json

```diff
@@ -2,28 +2,17 @@
   "version": "1.0.0-*",
   "description": "Updates the repos dependencies",
   "buildOptions": {
-    "emitEntryPoint": true
+    "emitEntryPoint": true,
+    "compile": [
+      "../../src/Microsoft.DotNet.Cli.Utils/DebugHelper.cs"
+    ]
   },
   "dependencies": {
-    "NETStandard.Library": "1.6.0",
-    "Microsoft.CSharp": "4.0.1",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
-    "System.Runtime.Serialization.Primitives": "4.1.1",
-    "Microsoft.DotNet.Cli.Build.Framework": {
-      "target": "project"
-    },
-    "NuGet.Versioning": "3.5.0-beta2-1484",
-    "Newtonsoft.Json": "9.0.1",
-    "Octokit": "0.18.0",
-    "Microsoft.Net.Http": "2.2.29"
+    "Microsoft.NETCore.App": "1.0.0",
```

Comment:
This is downgrading the Netstandard.Library version. Should it be consuming the version from core-setup master?

I don't think it matters, just wanted to call it out.